site2/website/versioned_docs/version-2.8.0/developing-binary-protocol.md

```diff
@@ -382,6 +382,29 @@ Parameters:
  * `validation_error` → *(optional)* Indicates that the consumer has discarded
    the messages due to: `UncompressedSizeCorruption`,
    `DecompressionError`, `ChecksumMismatch`, `BatchDeSerializeError`
+ * `properties` -> *(optional)* Reserved configuration items
```

Comment:
why would you like to use "->" instead of ":"?

Reply:
It's a copy&paste effort to inherit the content from the master branch.